doc/nrf/protocols/matter/getting_started/adding_clusters.rst

```diff
@@ -101,6 +106,11 @@ To edit clusters using the ZAP tool, complete the following steps:
 
       west zap-gui
 
+
+   .. note::
+   The ZAP tool's UI may vary between the ZAP tool versions. Following steps should taken as hints.
```

Comment:
```suggestion
   The ZAP tool's UI may vary between the ZAP tool versions. Following steps should be taken as hints.
```

Reply:
Will change event  for such sentence:
`The ZAP tool's UI may vary depending on the ZAP version. The following steps should be considered as guidelines.`